NEWS.md

```diff
@@ -25,7 +25,7 @@ See `vignette("in-packages")` for a detailed transition guide.
   request the previous approach.
 
 * `unnest_()`/`nest_()` and the lazyeval methods for `unnest()`/`nest()` are 
-  now defunct. They have been deprecated for some time, and since the interface
+  now defunct. They have been deprecated for some time, and, since the interface
```

Comment:
Would it be worth adding a link to the lifecycle package's [definitions](https://lifecycle.r-lib.org/articles/lifecycle.html) of deprecated and defunct?

Reply:
Yeah, that'd be great